core/src/test/resources/SpringXmlPropertyResolverTest.xml

```diff
@@ -15,7 +15,7 @@
 		name="testSpringXmlPropertyResolver"
 		class="java.lang.String"
 		>
-		<constructor-arg value="${jms.messageListenerClassName}"/>
```

Comment:
Deze test kijkt of een property via Spring ge-resolved wordt. Je kan hier het beste een andere property uitkiezen ( en de test een klein beetje aanpassen)